brage-import/src/test/java/no/sikt/nva/brage/migration/lambda/BrageEntryEventConsumerTest.java

```diff
@@ -1103,6 +1162,38 @@ private static void putAssociatedArtifactsToResourceStorage(NvaBrageMigrationDat
             });
     }
 
+    private Publication createCristinPublication(String cristinIdentifier) throws BadRequestException {
+        var publication = randomPublication().copy()
+                              .withAdditionalIdentifiers(
+                                  Set.of(new AdditionalIdentifier("Cristin", cristinIdentifier)))
+                              .withAssociatedArtifacts(List.of())
+                              .build();
+        return Resource.fromPublication(publication)
+                   .persistNew(resourceService, UserInstance.fromPublication(publication));
+    }
+
+    private Record createMinimalRecord(String cristinIdentifier) {
+        var minimalRecord = new Record();
+        var fakeDummyHandle = UriWrapper.fromUri("dymmy_handle_unis/1").getUri();
```

Comment:
Unis does not have handles we should be migrate. So a place holder is used. I intend to increment the number for each unis publication.